docs/artist/getCorrection.md

```diff
@@ -0,0 +1,104 @@
+Get the metadata and tracklist for an album on Last.fm using the album name or a musicbrainz id.
+
+No authentication required.
+
+## Parameters
+
+| Method        | Type                                                                                              | Default | Required                      | Description                                                                                                           |
+| ------------- | ------------------------------------------------------------------------------------------------- | ------- | ----------------------------- | --------------------------------------------------------------------------------------------------------------------- |
+| `artist`      | [string](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String) | `none`  | :white_check_mark:          | The artist which's album should be fetched.                                                                           |
+| `track`       | [string](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String) | `none`  | :white_check_mark:        | The track that should be fetched.                                                                                     |
+| `api_key`     | [token](https://www.last.fm/api/account/create)                                                   | `none`  | :white_check_mark:            | A Last.fm API key.                                                                                                    |
+
+## Responses
+
+Errors:
+
+- 2 : Invalid service - This service does not exist
+- 3 : Invalid Method - No method with that name in this package
+- 4 : Authentication Failed - You do not have permissions to access the service
+- 5 : Invalid format - This service doesn't exist in that format
+- 6 : Invalid parameters - Your request is missing a required parameter
+- 7 : Invalid resource specified
+- 8 : Operation failed - Something else went wrong
+- 9 : Invalid session key - Please re-authenticate
+- 10 : Invalid API key - You must be granted a valid key by last.fm
+- 11 : Service Offline - This service is temporarily offline. Try again later.
+- 13 : Invalid method signature supplied
+- 16 : There was a temporary error processing your request. Please try again
+- 26 : Suspended API key - Access for your account has been suspended, please contact Last.fm
+- 29 : Rate limit exceeded - Your IP has made too many requests in a short period
+
+!!! Warning
+    This API call returns 200 OK HTTP status codes even when the response contains an error.
+    
+## Attributes
+
+duration - in seconds

```

Comment:
I don't think this is an attribute that exists here? You can however include the other ones.